src/App.tsx

```diff
@@ -5,6 +5,11 @@ import "react-toastify/dist/ReactToastify.css";
 import FileEditorContainer from "./components/FileEditorContainer";
 import IndexingProgress from "./components/IndexingProgress";
 import InitialSetupSinglePage from "./components/Settings/InitialSettingsSinglePage";
+import posthog from "posthog-js";
+
+posthog.init("phc_xi4hFToX1cZU657yzge1VW0XImaaRzuvnFUdbAKI8fu", {
+  api_host: "https://us.i.posthog.com",
```

Comment:
a bit :-1: on this. I use ollama and a local tool to avoid sending anything to the internet.

Reply:
I hear you. Essentially it's your call.

Reply:
opt-out maybe?

Reply:
Yep thanks for the feedback @sebastian-philipp. We will implement opt-out too.